exporter/otlp/opentelemetry-exporter-otlp.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
   spec.required_ruby_version = '>= 2.5.0'
 
-  spec.add_dependency 'google-protobuf', '>= 3.4.1.1', '< 4'
+  spec.add_dependency 'google-protobuf', '>= 3.7.0', '< 4'
```

Comment:
If we don't require a specific patch level, I'd just use

```ruby
spec.add_dependency 'google-protobuf', '~> 3.7'
```

Reply:
Thanks, made the change to refer just by minor version and ignore patch level as you suggested.